x-pack/packages/kbn-elastic-assistant-common/impl/schemas/conversations/find_conversations_route.schema.yaml

```diff
@@ -91,6 +91,7 @@ paths:
                   message:
                     type: string
 
+  # TODO: No corresponding route/handler, can this be removed?
   /internal/elastic_assistant/conversations/current_user/_find:
```

Comment:
@YulNaumenko I couldn't find usage of this route -- are we okay to delete this? Seems there was a usage difference between `conversations/_find` and `conversations/current_user/_find` as we have no 'all-user' find, right?

Reply:
Yes, lets remove it.